pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +922,40 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA operations (PrepareResources and UnprepareResources).",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"operation_name"},
+	)
+
+	// DRAOperationsErrorsTotal is a counter that tracks the number of errors in the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsErrorsTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAOperationsErrorsTotalKey,
+			Help:           "Cumulative number of errors when preparing and unpreparing resources.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"operation_name"},
+	)
+
+	DRAGRPCOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAGRPCOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA GRPC operations",
```

Comment:
```suggestion
			Help:           "Duration in seconds of the DRA gRPC operations",
```

Reply:
@pohly Thank you for the review! I've committed your suggestion.